### DISCUSSION TITLE
新作业相关的代码好像没有更新

### DISCUSSION THREAD

**Discussion by Jaysama137** [BoomingTech/Piccolo#240] [General]
没找到相关文件
另外，作业里两个文件的路径是不是多写了一层runtime

**Comment by runck** [comment]
在作业3的branch里有

**Comment by Ol6rin** [comment]
是的，作业在专门的分支里 games104/homework03-animation-physics。
作业文档里面的路径确实存在错误，我们会尽快更新一版。

**Reply by JiangYi07** [reply]
作业分支和main分支区别好像不止作业文档里面提到的那些类，直接在main上写作业现实吗。

**Reply by Ol6rin** [reply]
> 作业分支和main分支区别好像不止作业文档里面提到的那些类，直接在main上写作业现实吗。

作业分支的环境是专门为作业准备的，main分支并不能满足这个需求。
